src/circuit/hash.rs

```diff
@@ -130,9 +95,14 @@ fn hash_node(circ: &impl HugrView, node: Node, state: &mut HashState) -> u64 {
     let op = circ.get_optype(node);
     let mut hasher = FxHasher64::default();
 
+    // Hash the node children
     if circ.children(node).count() > 0 {
-        panic!("Cannot hash container node {node:?}.");
+        let container: SiblingGraph =
+            SiblingGraph::try_new(circ, node).expect("Cannot hash container node");
+        container.circuit_hash().hash(&mut hasher);
```

Comment:
Just in passing but I have to say - wtf, the method is `u64.hash(&mut hasher)` not `FxHasher64.hash(u64)` ?!